mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -57,55 +67,46 @@ def ocn_modelvsobs(config, field):
     outputTimes = config.getExpression(field.lower() + '_modelvsobs',
                                        'comparisonTimes')
 
-    f = netcdf_dataset(meshfile,mode='r')
+    f = netcdf_dataset(meshfile, mode='r')
     lonCell = f.variables["lonCell"][:]
     latCell = f.variables["latCell"][:]
 
-    if field.lower() == 'mld':
-        obs_filename = "%s/holtetalley_mld_climatology.nc" % obsdir
+    varList = [field]
 
-        ds = xr.open_mfdataset(infiles, preprocess=lambda x: 
-                               preprocess_mpas(x, yearoffset=yr_offset,
-                                               timestr=['xtime_start', 'xtime_end'],
-                                               onlyvars=['time_avg_dThreshMLD']))
-        ds = remove_repeated_time_index(ds)
-        ds.rename({'time_avg_dThreshMLD':'mpasData'}, inplace = True)
+    if field == 'mld':
 
-        #Load MLD observational data
+        selvals = None
+
+        # Load MLD observational data
+        obs_filename = "{}/holtetalley_mld_climatology.nc".format(obsdir)
         dsData = xr.open_mfdataset(obs_filename)
 
-        #Increment month value to be consistent with the model output
+        # Increment month value to be consistent with the model output
         dsData.iMONTH.values += 1
 
-        #Rename the time dimension to be consistent with the SST dataset
-        dsData.rename({'month':'calmonth'}, inplace = True)
-        dsData.rename({'iMONTH':'month'}, inplace = True)
+        # Rename the time dimension to be consistent with the SST dataset
+        dsData.rename({'month': 'calmonth'}, inplace=True)
+        dsData.rename({'iMONTH': 'month'}, inplace=True)
 
-        #rename appropriate observational data for compactness
-        dsData.rename({'mld_dt_mean':'observationData'}, inplace = True)
+        obsFieldName = 'mld_dt_mean'
 
-        #Reorder dataset for consistence
+        # Reorder dataset for consistence
         dsData = dsData.transpose('month', 'iLON', 'iLAT')
 
-        #Set appropriate MLD figure labels
+        # Set appropriate MLD figure labels
         obsTitleLabel = "Observations (HolteTalley density threshold MLD)"
         fileOutLabel = "mldHolteTalleyARGO"
         unitsLabel = 'm'
 
-    elif field.lower() == 'sst':
+    elif field == 'sst':
 
-        ds = xr.open_mfdataset(infiles, preprocess=lambda x: 
-                               preprocess_mpas(x, yearoffset=yr_offset,
-                                               timestr=['xtime_start', 'xtime_end'],
-		                               onlyvars=['time_avg_activeTracers_temperature'],
-		                               selvals={'nVertLevels':1}))
-        ds = remove_repeated_time_index(ds)
-        ds.rename({'time_avg_activeTracers_temperature':'mpasData'}, inplace = True)
+        selvals = {'nVertLevels': 1}
```

Comment:
@milenaveneziani or @vanroekel, could you comment on this.  I would agree that it's likely a bug and we're looking at the temperature in layer 1 instead of layer 0.  But this needs to be reported as an issue and addressed with a separate PR.